charts/rollup/files/genesis/geth-genesis.json

```diff
@@ -16,11 +16,17 @@
         "ethash": {},
         {{- if .Values.config.rollup.genesis.extraDataOverride }}
         "astriaExtraDataOverride": "{{ .Values.config.rollup.genesis.extraDataOverride }}",
-        {{ end -}}
+        {{- end }}
         "astriaOverrideGenesisExtraData": {{ .Values.config.rollup.genesis.overrideGenesisExtraData }},
         "astriaSequencerInitialHeight": {{ .Values.config.sequencer.initialBlockHeight }},
+        {{- if not .Values.global.dev }}
         "astriaDataAvailabilityInitialHeight": {{ .Values.config.celestia.initialBlockHeight }},
         "astriaDataAvailabilityHeightVariance": {{ .Values.config.celestia.heightVariance }}
+        {{- else }}
+        "astriaRollupName": "{{ .Values.config.rollup.name }}",
+        "astriaCelestiaInitialHeight": {{ .Values.config.celestia.initialBlockHeight }},
+        "astriaCelestiaHeightVariance": {{ .Values.config.celestia.heightVariance }}
+        {{- end }}
```

Comment:
Note pattern used throughout for configuration which might change between releases.